js/views/modals/Settings/Page.js

```diff
@@ -228,7 +229,8 @@ export default class extends baseVw {
         headerMinWidth: this.headerMinWidth,
       }));
 
-      this.$formFields = this.$('select[name], input[name], textarea[name]');
+      const formFields = 'select[name], input[name], textarea[name], div[contenteditable][name]';
```

Comment:
One thing that's not obvious is you have to update the list of elements that are collected when saving to include the trumbowyg field.